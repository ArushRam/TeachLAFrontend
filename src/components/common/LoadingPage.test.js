import React from "react";
import { shallow } from "enzyme";
import LoadingPage from "./LoadingPage";

describe("LoadingPage", () => {
  it("should render correctly", () => {
    const component = shallow(<LoadingPage />);

    expect(component).toMatchSnapshot();
  });
});
