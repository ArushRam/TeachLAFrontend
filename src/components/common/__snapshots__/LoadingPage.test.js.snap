// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingPage should render correctly 1`] = `
<div
  className="Loading"
>
  <div
    className="Loading-title"
  >
    Loading
  </div>
  <onlyUpdateForKeys(Loader)
    color="#171124"
    loading={true}
    size={250}
    sizeUnit="px"
  />
</div>
`;
